_targets.R

```diff
@@ -16,14 +16,14 @@ tar_plan(
   tar_target(
     example_shapefile,
     get_example_shapefile(),
-    format = format_shapefile
+    format = format_geoparquet
   ),
 
   country_codes = country_codes(query = "Australia"),
 
   tar_target(
     example_gadm,
```

Comment:
```suggestion
    example_gadm.zip,
```

The main trick to using /vsizip/ is the target name (and thus the file the target is stored in) has to end with ".zip". 

This naming affects both writing and reading, mainly because the code as written expects a .shp file in the project directory to ZIP and cleanup after. The code could be altered to work with a subdirectory, but you still need the .zip in the resulting target object stored for subsequent reading (or GDAL will not recognize it as a zip file source)

On first run of your code GDAL creates `"example_gadm"` folder with `"example_gadm.shp"` within it, resulting in `Layer 'example_gadm' already exists (GDAL error 1)` after running the pipeline at least once.